src/app_service/service/community_tokens/async_tasks.nim

```diff
@@ -227,20 +236,86 @@ type
 const fetchCollectibleOwnersTaskArg: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
   let arg = decode[FetchCollectibleOwnersArg](argEncoded)
   try:
-    let response = collectibles.getCollectibleOwnersByContractAddress(arg.chainId, arg.contractAddress)
+    var response = collectibles.getCollectibleOwnersByContractAddress(arg.chainId, arg.contractAddress)
+
+    var owners = fromJson(response.result, CollectibleContractOwnership).owners
+    owners = owners.filter(x => x.address != ZERO_ADDRESS)
+
+    response = communities_backend.getCommunityMembersForWalletAddresses(arg.communityId, arg.chainId)
```

Comment:
... and here :)

Reply:
callPrivateRPC will raise an exception if there is a response error